test/reshape.jl

```diff
@@ -419,23 +420,25 @@ end
     @test_throws ArgumentError flatten(df_bad, [:b, :c])
 end
 
-@testset "test RepeatedVector for categorical" begin
-    v = categorical(["a", "b", "c"], ordered=true)
-    levels!(v, ["b", "c", "a"])
-    rv = DataFrames.RepeatedVector(v, 1, 1)
-    @test isordered(v)
-    @test isordered(categorical(v))
-    @test levels(v) == ["b", "c", "a"]
-    @test levels(categorical(v)) == ["b", "c", "a"]
-
-    v = categorical(["a", "b", "c"])
-    levels!(v, ["b", "c", "a"])
-    rv = DataFrames.RepeatedVector(v, 1, 1)
-    @test !isordered(v)
-    @test !isordered(categorical(v))
-    @test levels(v) == ["b", "c", "a"]
-    @test levels(categorical(v)) == ["b", "c", "a"]
-end
+# TODO: uncomment these tests when we improve handling of categorical here
```

Comment:
At least do tests on `categorical(rv)` pass if you drop test on `rv` itself?

Reply:
They will not pass. This is the point that `categorical(rv)` forgets about the ordering:
```
julia> isordered(v)
true

julia> levels(v)
3-element Array{String,1}:
 "b"
 "c"
 "a"

julia> isordered(categorical(rv))
false

julia> levels(categorical(rv))
3-element Array{String,1}:
 "a"
 "b"
 "c"
```

Reply:
Ah, right. That would require a new function in DataAPI to know when `levels` is O(1). Then we could always preserve the order of levels, and `isordered`. But that's quite low priority.

Reply:
At any rate probably better remove these tests.